plugins/packages/polygon/main.go

```diff
@@ -188,5 +276,37 @@ func Migrate(db *sql.DB, chainid uint64) error {
 		}
 		log.Info("bor migrations done")
 	}
+	if schemaVersion < 2 {
+		log.Info("Inside bor migration v2", "time", time.Now())
+		rows, _ := db.QueryContext(context.Background(), "SELECT number, extra FROM blocks.blocks")
```

Comment:
We want to limit this to blocks where the coinbase is currently the 0 address. I'd change this to:

```
SELECT number, extra from blocks.blocks where coinbase = X'00';
```